README.md

```diff
@@ -0,0 +1,87 @@
+### Safepay React Native Demo
+
+This project aims to be the easiest way for our merchants to consume safepay-checkout in their React Native applications.
+
+### Installation
+
+`npm:`
+
+> #### $ npm install
+
+`ios:`
+
+> #### $ pod install
+
+#### `npm start`
+
+Runs your app in development mode.
+
+#### `npm run ios`
+
+Like npm start, but also attempts to open your app in the iOS Simulator if you're on a Mac and have it installed.
+
+#### `npm run android`
+
+Like npm start, but also attempts to open your app on a connected Android device or emulator. Requires an installation of Android build tools (see [React Native](https://reactnative.dev/docs/getting-started) docs for detailed setup).  
+
+#### Usage
+
+#### SafepayCheckout
+
+`SafepayCheckout` component is the main component which wraps everything and provides a couple of props (see Config below).
+
+#### Example
+
+``` ts
+import React from 'react';
+import SafepayCheckout from 'safepay-react-native';
```

Comment:
This import needs to be updated, also environment import is also missing
Use this:
import SafepayCheckout, {environment, theme}  from '@sfpy/react-native-sdk';